ansys-grantami-serverapi-openapi/pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry]
 name = "ansys-grantami-serverapi-openapi"
 description = "Autogenerated client library for the Granta MI Server API."
-version = "3.0.0.dev260"
+version = "3.0.0.dev266"
```

Comment:
Does this version number need to be manually bumped to 3.0.1 as well?

Reply:
I'll do that in a separate PR. We should probably bump main to 4.0.0

Reply:
#201